homeassistant/components/bmp280/sensor.py

```diff
@@ -51,13 +51,14 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
         # this usually happens when the board is I2C capable, but the device can't be found at the configured address
         if str(error.args[0]).startswith("No I2C device at address"):
             _LOGGER.error(
-                "%s. Hint: Check wiring and make sure that the SDO pin is tied to either ground (0x76) or VCC (0x77)!",
+                "%s. Hint: Check wiring and make sure that the SDO pin is tied to either ground (0x76) or VCC (0x77)",
                 error.args[0],
             )
             raise PlatformNotReady()
-        raise error
+        _LOGGER.error(error.args[0])
```

Comment:
Can we just pass the exception as is as log message?
```suggestion
        _LOGGER.error(error)
```

Reply:
Sure, thanks